### PR TITLE
docs: update TGW firewall user steps

### PR DESCRIPTION
**Pull request by SanketD92**
Issue #, if available:
N/A

Description of changes:
Adding details for TGW and firewall instructions.

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/src/templates/README.md [hunk 1]
```diff
@@ -1,30 +1,48 @@
...
 
-This template was originally designed for use with Palo Alto Networks' VM Series firewall offering on AWS Marketplace. However, the template can be modified to leverage any firewall offering on AWS Marektplace that has an AMI.
+----
 
-### onboard-account-tgw.cfn.yaml
-A template to help hosting/member accounts integrate with the Network account. 
+## Onboard Account (Bring Your Own Network)
+This [template]((./onboard-account-byon.cfn.yaml)) does everything onboard-account.cfn.yaml does, except allows the user to bring their own network components (VPC and subnets). This template therefore gives the user more granular control by using pre-configured networks for hosting workspaces. These pre-confurations can include, but are not limited to, network traffic monitoring, cross account network control (using route tables, [VPC peering](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/peer-with-vpc-in-another-account.html), [PrivateLink](https://docs.aws.amazon.com/vpc/latest/userguide/endpoint-services-overview.html) etc.), and additional security layers.
```
Comment (maghirardelli) [inline solutions/swb-reference/src/templates/README.md:13]: confurations --> configurations typo

File: solutions/swb-reference/src/templates/README.md [hunk 2]
```diff
@@ -1,30 +1,48 @@
...
+
+----
+## Integrating with a Central Network Account
+The following CFN templates help setting up hosting accounts while integrating them with a Central Network Account, using Transit Gateway and Network Firewall.
+## Firewall and TGW
+A Central Network account could be available with some customers who have specific network monitoring and regulatory requirements to be carried out with that account, thereby allowing the user manage main account and hosting account network traffic all at one place.
```
Comment (maghirardelli) [inline solutions/swb-reference/src/templates/README.md:19]: get rid of "thereby" it does nothing.

and "allowing the user **to** manage..."

File: solutions/swb-reference/src/templates/README.md [hunk 3]
```diff
@@ -1,30 +1,48 @@
...
+
+----
+## Integrating with a Central Network Account
+The following CFN templates help setting up hosting accounts while integrating them with a Central Network Account, using Transit Gateway and Network Firewall.
+## Firewall and TGW
+A Central Network account could be available with some customers who have specific network monitoring and regulatory requirements to be carried out with that account, thereby allowing the user manage main account and hosting account network traffic all at one place. 
+
+This [template](./new-firewall-vpc.yaml) can be deployed in such a central network account for that purpose. This uses a network firewall AMI and creates a Security VPC, Gateway Load Balancer with firewall instance targets, and a Transit Gateway. The Security VPC can be used for monitoring network traffic centrally between main account and hosting account(s).
+
+This template was originally designed for use with Palo Alto Networks' VM Series firewall offering on AWS Marketplace. However, the template can be modified to leverage y firewall offering on AWS Marektplace that has an AMI.
```
Comment (maghirardelli) [inline solutions/swb-reference/src/templates/README.md:23]: "leverage y firewall" --> "leverage any firewall"?

File: solutions/swb-reference/src/templates/README.md [hunk 4]
```diff
@@ -1,30 +1,48 @@
...
+As is, these templates depend on a subscription to an EC2 firewall appliance. We recommend you or your SA review the network settings in both the `new-firewall-vpc.yaml` file and the `onboard-account-tgw.cfn.yaml` to update them to best suit your needs.
 
 ### How to use these:
-1. Launch the stack in a network account
+1. Launch the stack in a central network account
 1. Share the network account's transit gateway with the hosting account
-    1.Go to `VPC` -> `Transit Gateways` -> `Transit Gateways`
+    1. Go to `VPC` -> `Transit Gateways` -> `Transit Gateways`
     1. Select the transit gateway and, from the Actions menu, select "Share transit gateway" 
     1. If a resource share doesn't already exist for your TGW, then create one. Make sure to give access to the app account on the "Grant access to principals"
```
Comment (maghirardelli) [inline solutions/swb-reference/src/templates/README.md:36]: "app account"?

File: solutions/swb-reference/src/templates/README.md (hunk 1 above)
Comment (aws-tyler) [reply solutions/swb-reference/src/templates/README.md:13]: I'd relpace `onboard-account.cfn.yaml` with a link to the `Onboard Account` section above

File: solutions/swb-reference/src/templates/README.md [hunk 5]
```diff
@@ -1,30 +1,48 @@
...
+Here are a few onboarding template choices that you could use for onboarding your hosting account. The difference between each of these templates is based on the level of network flexibility you need. This is where your SWB compute resources would be hosted.
 
-## onboard-account-byon.cfn.yaml
-A CFN template that does everything onboard-account.cfn.yaml does, except allows the hosting account to use its own VPCs.
+----
 
-## The following are CFN templates that help setting up networks for transit gate ways in the BYON scenario
-### new-firewall-vpc-TGW.yaml
-A template to help a Network account set up VPCs, a firewall, and a Transit Gateway. 
+## Onboard Account
```
Comment (aws-tyler) [inline solutions/swb-reference/src/templates/README.md:7]: Might want to add `Default` somewhere in here so its easy to refer back to when we're talking about other types of onboarding methods.

File: solutions/swb-reference/src/templates/README.md [hunk 6]
```diff
@@ -1,30 +1,48 @@
...
+----
 
-### onboard-account-tgw.cfn.yaml
-A template to help hosting/member accounts integrate with the Network account. 
+## Onboard Account (Bring Your Own Network)
+This [template]((./onboard-account-byon.cfn.yaml)) does everything onboard-account.cfn.yaml does, except allows the user to bring their own network components (VPC and subnets). This template therefore gives the user more granular control by using pre-configured networks for hosting workspaces. These pre-confurations can include, but are not limited to, network traffic monitoring, cross account network control (using route tables, [VPC peering](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/peer-with-vpc-in-another-account.html), [PrivateLink](https://docs.aws.amazon.com/vpc/latest/userguide/endpoint-services-overview.html) etc.), and additional security layers.
+
+----
+## Integrating with a Central Network Account
+The following CFN templates help setting up hosting accounts while integrating them with a Central Network Account, using Transit Gateway and Network Firewall.
```
Comment (aws-tyler) [inline solutions/swb-reference/src/templates/README.md:17]: CFN probably needs to be completely spelled out

Comment (aws-tyler) [reply solutions/swb-reference/src/templates/README.md:17]: I don't think you need the comma between `Account` and `using`

File: solutions/swb-reference/src/templates/README.md (hunk 2 above)
Comment (aws-tyler) [reply solutions/swb-reference/src/templates/README.md:19]: `A Central Network account could be available with some customers who have`
to
`A Central Network account can be used by customers who have` ?

File: solutions/swb-reference/src/templates/README.md [hunk 7]
```diff
@@ -1,30 +1,48 @@
...
+## Firewall and TGW
+A Central Network account could be available with some customers who have specific network monitoring and regulatory requirements to be carried out with that account, thereby allowing the user manage main account and hosting account network traffic all at one place. 
+
+This [template](./new-firewall-vpc.yaml) can be deployed in such a central network account for that purpose. This uses a network firewall AMI and creates a Security VPC, Gateway Load Balancer with firewall instance targets, and a Transit Gateway. The Security VPC can be used for monitoring network traffic centrally between main account and hosting account(s).
```
Comment (aws-tyler) [inline solutions/swb-reference/src/templates/README.md:21]: `between the main and hosting accounts`

File: solutions/swb-reference/src/templates/README.md [hunk 8]
```diff
@@ -1,30 +1,48 @@
...
+
+This template was originally designed for use with Palo Alto Networks' VM Series firewall offering on AWS Marketplace. However, the template can be modified to leverage y firewall offering on AWS Marektplace that has an AMI.
+
+## Onboard Account TGW
+This [template](./onboard-account-tgw.cfn.yaml) onboards a hosting account as well as helps integrate with the resources created as part of the `new-firewall-vpc.yaml` template deployed in the Central Network account. You need to provide the Gateway Load Balancer Service ID and Transit Gateway ID output values from the Central Network account to this template's input parameters. This template provisions network components (VPC and Subnets) based on the public/private nature selected during stack creation.
```
Comment (aws-tyler) [inline solutions/swb-reference/src/templates/README.md:26]: Link to the section instead of `new-firewall-vpc.yaml`?

Comment (aws-tyler) [reply solutions/swb-reference/src/templates/README.md:26]: `public/private nature`: Not sure what that means?

File: solutions/swb-reference/src/templates/README.md [hunk 9]
```diff
@@ -1,30 +1,48 @@
...
-As is, these templates depend on a subscription to an EC2 firewall appliance. We recommend you or your SA review the network settings in both the `new-firewall-vpc.yaml` file and the `onboard-account-tgw.cfn.yaml` to update them to best suit your needs. This may include removing the EC2 firewall and the Gatewayload balancer, and updating the Transit Gateway's and onboarding account's route tables to reroute trafic that used to go through the EC2 firewall.
+As is, these templates depend on a subscription to an EC2 firewall appliance. We recommend you or your SA review the network settings in both the `new-firewall-vpc.yaml` file and the `onboard-account-tgw.cfn.yaml` to update them to best suit your needs.
 
 ### How to use these:
-1. Launch the stack in a network account
+1. Launch the stack in a central network account
 1. Share the network account's transit gateway with the hosting account
-    1.Go to `VPC` -> `Transit Gateways` -> `Transit Gateways`
+    1. Go to `VPC` -> `Transit Gateways` -> `Transit Gateways`
     1. Select the transit gateway and, from the Actions menu, select "Share transit gateway"
```
Comment (aws-tyler) [inline solutions/swb-reference/src/templates/README.md:35]: I know you didn't write this part, but I'd split this up into two separate actions: selecting the gateway and selecting "share"

File: solutions/swb-reference/src/templates/README.md (hunk 4 above)
Comment (aws-tyler) [reply solutions/swb-reference/src/templates/README.md:36]: `hosting` is right, I think?

File: solutions/swb-reference/src/templates/README.md [hunk 10]
```diff
@@ -1,30 +1,48 @@
...
-1. On the onboarding account, accept the resource share
+1. On the hosting account, accept the resource share
     1. Resource Access Manager -> Shared with me -> Accept the invite
-1. Launch the onboarding stack (`onboard-account-tgw.cfn.yaml`) in the hosting account with the relevant outputs from the network account onboarding
-1. Use the outputs from the onboarding stack to onboard the account with the main account per the instructions in the [setup instructions](../../README.md#setup-instructions-for-swbv2p1).
+1. Launch the onboarding stack (`onboard-account-tgw.cfn.yaml`) in the hosting account with the relevant outputs from the central network account stack.
+1. Since this stack creates its own network components, associate its subnets to Transit Gateway by adding them to the SecurityVPC route table. This helps centrally inspect egress traffic inspection.
+1. Create Gateway Load Balancer endpoints in the hosting account. Each endpoint needs a subnet per AZ. The Internet Gateway route table needs to have these endpoints added for ingress traffic inspection. (Ignore this step if using private hosting account subnets)
```
Comment (aws-tyler) [inline solutions/swb-reference/src/templates/README.md:41]: Availability Zone instead of AZ?

Comment (aws-tyler) [reply solutions/swb-reference/src/templates/README.md:41]: I would move the `ignore this step` condition to the front. Maybe just say "If you are using public hosting account subnets, ..."

**Review by aws-tyler — APPROVED** [review]

File: solutions/swb-reference/src/templates/README.md (hunk 9 above)
Comment (SanketD92) [reply solutions/swb-reference/src/templates/README.md:35]: Keeping as-is for now.

**Review by maghirardelli — APPROVED** [review]

**Pull request merged**